test/NameRegistry.t.sol

```diff
@@ -2157,7 +2159,9 @@ contract NameRegistryTest is Test {
         nameRegistry.completeRecovery(ALICE_TOKEN_ID);
 
         assertEq(nameRegistry.balanceOf(alice), 1);
-        if (alice != notRecovery) assertEq(nameRegistry.balanceOf(notRecovery), 0);
+        if (alice != notRecovery) {
+            assertEq(nameRegistry.balanceOf(notRecovery), 0);
```

Comment:
a lot of these one-liners are now being expanded to multi-liners (see NameRegistry.sol changes). is this some new requirement imposed by forge fmt? otherwise, would like to exclude these from the pr to make it easier to review

Reply:
This happened because of some auto-lint in my local VSCode. I amended the commit to remove this inadvertent change.

Reply:
nit: this one still isn't resolved